icons/equinixmetal.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Equinix Metal</title><path d="M9.6 5.288v12.066L8 16.819V5.825L3.2 7.431v7.783l-1.6-.537V7.965L0 8.5v7.266l4.8 1.606V8.519l1.6-.535v9.924l4.8 1.606V6.378l.8-.268.8.268v13.136l4.8-1.606V7.984l1.6.535v8.854l4.8-1.606V8.501l-1.6-.537v6.713l-1.6.537V7.43L16 5.825v10.994l-1.601.535V5.288L12 4.486z"/></svg>
```

Comment:
I'm seeing some significant differences when comparing your SVG to [the source SVG you mentioned in the issue](https://upload.wikimedia.org/wikipedia/en/a/a7/Equinix.svg) (see below). Did you use that SVG as a source, or a different one?

On an unrelated note, it looks like you used a precision greater than `3` with SVGO, however the icon doesn't look complex enough to require that. Did you try a lower precision?

![compare svgs](https://user-images.githubusercontent.com/3742559/127717567-1ab173c7-4b66-42cc-94bc-3a3119c2acdd.png)

Reply:
yes i used this as source! just cropped the name out. 

I think i set it yo 4 not sure…

Reply:
Thank you for confirming. I compared once more, this time to both the Wikimedia version and the SVG found in the header of the website and I'm getting the same comparison as above for both. It seems something went wrong when you cropped it.

Could you try this again? And this time using the SVG in the header of the website as a starting point (since 1) it's more official and 2) that's also where the hex value comes from). Be sure to keep the proportions of the logo constant when you resize it and try to set the precision of SVGO(MG) to 3 this time 🙂 

if you're having any trouble, don't hesitate to ask for help here. Please share the graphics editor you use so we can help you better.